docs/signatory-architecture.md

```diff
@@ -0,0 +1,295 @@
+---
+id: architecture
+title: Signatory Architecture
+sidebar_label: Signatory Architecture
+---
+
+# Signatory C4 Architecture Model 
+
+Signatory is the perfect solution for secure and reliable remote signing in Tezos Blockchain cases. With easy-to-use configuration options, users can set up vaults and policies to ensure that only authorized operations are signed. Signatory also supports a variety of hardware-based and cloud-based HSMs, such as [AWS KMS](https://aws.amazon.com/kms/) and [YubiHSM](https://www.yubico.com/?utm_source=google&utm_medium=pd:search&utm_campaign=US_B2C_LeadGen_Google_SEM_Brand&utm_content=&gclid=CjwKCAiAmJGgBhAZEiwA1JZolnAOjpSi_GVY8qz7NyLHASwkXHBu2t1aPNWl2WWHl4Nb2S19OsAWMBoCGucQAvD_BwE), to protect cryptographic keys.
```

Comment:
"perfect"? the architecture diagram page shouldn't read like a sales and marketing material.